.github/workflows/ci.yml

```diff
@@ -18,11 +18,9 @@ jobs:
       - uses: actions/checkout@v2
 
       - name: nvm install
-        shell: bash -l {0}
         run: nvm install
 
       - name: nvm use
```

Comment:
You could probably go further and get rid of this step completely as we call nvm use before the install and build commands anyway so this is redundant